### DISCUSSION TITLE
【记录】成为传奇绳匠的第二天

### DISCUSSION THREAD

**Discussion by TheVoidEnd** [share121/inter-knot#4506] [灌水]
这里是终焉～
前几天通过了以太适性检测之后，就一直在准备绳匠和盗洞客的东西了，就是还不知道哪里的空洞比较安全，或许会去六分街那边看看？听说风评不错来着
**好用的萝卜，趁手的武器，还有应急用的丁尼…**
音擎好贵的说…，对我这种新手绳匠来说有必要吗？
![IMG_20240907_003634](https://github.com/user-attachments/assets/38c8c2ca-5d31-43f8-a456-1737639d9f9a)
各位前辈还有什么建议喵？

**Comment by XMSTRIKEGP** [comment]
不必担心，总会有的(ง •̀_•́)ง‼
（当你没找到默契的伙伴时，就有可能找到优秀的引擎，冒紫光那种）

**Comment by Uknown-wang** [comment]
这个不是六分街玩具店买的玩具吗？
